src/Messaging/Mail/MailBag.php

```diff
@@ -302,6 +333,18 @@ function jsonSerialize(): array
             'recipients' => $recipients
         ];
 
+        $replyTo = [];
+        if ($this->getReplyToEmail()) {
```

Comment:
Here is one mistake, if replyTo email address is not set then code failed with message:
```
PHP Fatal error:  Uncaught Error: Call to a member function getEmail() on null in /Users/suchy/project/php-api-client-calthaeu/src/Messaging/Mail/MailBag.php:187
```

because we always want to retrieve property by method on object that was not initialize:
```
public function getReplyToEmail(): string
    {
        return $this->replyTo->getEmail();
    }
```

To fix code should look like this:
```
/**
     * @return string
     */
    public function getReplyToEmail(): ?string
    {
        if ($this->replyTo) {
            return $this->replyTo->getEmail();
        }

        return null;
    }

    /**
     * @return string
     */
    public function getReplyToName(): ?string
    {
        if ($this->replyTo) {
            return $this->replyTo->getName();
        }

        return null;
    }
```

and then it works perfectly

Reply:
ahh, indeed. Fixed. Thanks you!